src/items.cpp

```diff
@@ -2439,3 +2439,145 @@ void createCustomInventory(Stat* stats, int itemLimit)
 		}
 	}
 }
+
+node_t* itemNodeInInventory(Stat* myStats, Item* itemToFind, Category cat)
```

Comment:
This either needs a check to ensure that the function is not run when itemToFind is `nullptr`, or, better yet, make myStats and itemToFind references rather than pointers.